docs/references/context/page.md

```diff
@@ -28,32 +32,41 @@ parts of the request.
   // Consider the path to be /employee/{id}
   id := ctx.Request.PathParam("id")
   ```
-- `Bind(interface{})` - to access a decoded format of the request body, the body is mapped to the interface provided 
+- `Bind(interface{})` - to access a decoded format of the request body, the body is mapped to the interface provided
+
   ```go
-  // incoming request body is 
+  // incoming request body is
   // {
   //    "name" : "trident",
   //    "category" : "snacks"
   // }
-  
-  type product struct{
+
+   type product struct{
       Name string `json:"name"`
       Category string `json:"category"`
-  }
+   }
 
-  var p product 
+  var p product
   ctx.Bind(&p)
   // the Bind() method will map the incoming request to variable p
   ```
+
 - `HostName()` - to access the host name for the incoming request
   ```go
-  // for example if request is made from xyz.com
-  host := ctx.Request.HostName()
-  // the host would be http://xyz.com
-  // Note: the protocol if not provided in the headers will be set to http by default
-  ``` 
-  
+    // for example if request is made from xyz.com
+    host := ctx.Request.HostName()
+    // the host would be http://xyz.com
+    // Note: the protocol if not provided in the headers will be set to http by default
+  ```
+- `Params()` - to access all query parameters for a given key returning comma-separated values
```

Comment:
this seems wrong as we are also support a=b&a=c

Reply:
updated the documentation and godoc comment